src/ceph-volume/ceph_volume/tests/util/test_disk.py

```diff
@@ -250,138 +166,88 @@ def test_float(self):
 class TestGetDevices(object):
 
     def setup_paths(self, tmpdir):
-        paths = []
-        for directory in ['block', 'dev', 'mapper']:
-            path = os.path.join(str(tmpdir), directory)
-            paths.append(path)
-            os.makedirs(path)
-        return paths
-
-    def test_no_devices_are_found(self, tmpdir):
-        result = disk.get_devices(
-            _sys_block_path=str(tmpdir),
-            _dev_path=str(tmpdir),
-            _mapper_path=str(tmpdir))
-        assert result == {}
+        path = os.path.join(str(tmpdir), 'block')
+        os.makedirs(path)
+        return path
```

Comment:
if this is `setup_paths` but it is returning a `path`, it seems that the function isn't doing what it implies, I would recommend updating the name of the function to reflect the change